source/js/views/serverLogs.js

```diff
@@ -22,10 +23,10 @@ exports = module.exports = internals.ServerLogsView = Backbone.View.extend({
 
     initialize: function () {
 
-        var self = this;
-        this.listenTo(this.collection, 'add', function (model) {
+        const self = this;
+        this.listenTo(this.collection, 'add', (model) => {
 
-            this.render();
+            self.this.render();
```

Comment:
Probably a typo here ? You actually need to keep it a normal function if you need its `this`.